farm.go

```diff
@@ -18,3 +25,11 @@ func main() {
 		log.Fatal("ListenAndServe: ", err)
 	}
 }
+
+func connectDB(c *app.Context) {
+	var err error
+	c.DB, err = sql.Open("postgres", "user=postgres dbname=farm_development sslmode=disable")
```

Comment:
So, that's pretty clever but I wouldn't do it. What happens if despite returning an error the `db.Open` call returns something awkward which gets assigned to the `c.DB`? I'd do it in a more mundane, boring way:

```
db, err := sql.Open("postgres", "user=postgres dbname=farm_development sslmode=disable")
if err != nil {
  log.Fatal(err)
}
c.DB = db
```